Lib/test/support/__init__.py

```diff
@@ -1778,13 +1772,6 @@ def requires_resource(resource):
     else:
         return unittest.skip("resource {0!r} is not enabled".format(resource))
 
-def requires_android_level(level, reason):
```

Comment:
I suggest to keep it. I'm in touch with a developer who wants to support older Android versions, and in the future, we may have to use it again for features only working on newer Android versions.

Reply:
I prefer not to clutter ``test.support`` with dead code and it is easy to restore ``requires_android_level`` with ``git show  <sha1> | git apply -R``.